insights/parsers/ssh.py

```diff
@@ -176,6 +179,52 @@ def get_line(self, keyword, default=''):
                 keyword=keyword, value=default
             )
 
+    def does_line_use_plus(self, keyword):
```

Comment:
Since this is applies to keywords, and not just to lines, I would recommend this be called `plus_in_values_of` or `keyword_contains_plus`.

Reply:
I prefer `keyword_contains_plus` of the two suggestions, and I think `does_keyword_use_plus` is also pretty good.

Reply:
The "+" symbol with the meaning denoted by the comments can occur only in one place in an sshd.conf line. It really applies to the whole line at once and not to any individual value on its own. Of your suggestions, `keyword_contains_plus` is the most near being correct, but again, it's not the *keyword* that *contains* a "+", it's the *line* that contains it at a *very specific* place. So if you deem the current name `does_line_use_plus` unfit, I can change it either to `line_contains_plus` or `line_uses_plus` or `keyword_uses_plus` - please pick your choice. Please see the comments and the OpenSSH documentation for information how the "+" works.